elf_size_analyze/tree_node.py

```diff
@@ -0,0 +1,104 @@
+"""
```

Comment:
I'd rename the file to `tree.py` as this is an implementation of a tree data structure. The class would stay as `TreeNode` as it represents a single node of a tree (and the root node is just the whole tree).